etc/config/analysis.amazon.properties

```diff
@@ -0,0 +1,14 @@
+compilers=&llvm-mca
+compilerType=analysis
+supportsBinary=false
```

Comment:
The properties files have a hierarchical structure. The default `{LANG}.default.properties` being a base from where the rest of the levels override from. I suggest you move the common properties for all versions of this to a `analysis.default.properties` so when locally run, you don't have to tweak the files by hand

Reply:
These loose properties are ok this way, though? (compilerType and supportsBinary, for example)
Or should I put them under the llvm-mca group?

Moved this file to analysis.defaults.properties.